formula-android/src/main/java/com/instacart/formula/android/Feature.kt

```diff
@@ -12,20 +14,43 @@ import io.reactivex.rxjava3.core.Observable
  *
  * Take a look at [FeatureFactory] for more information.
  */
-class Feature private constructor(
-    val stateObservable: Observable<Any>,
-    val viewFactory: ViewFactory<Any>
-) {
+sealed class Feature {
```

Comment:
Making this into a sealed class to support `RxJavaFeature` and `StateFlowFeature`